examples/opentelemetry/docker-compose.yaml

```diff
@@ -63,9 +63,9 @@ services:
     healthcheck:
       test: ["CMD-SHELL", "curl -sf http://localhost:13133 || exit 1"]
       interval: 1s
-      timeout: 120s
+      timeout: 10s
       retries: 120
-      start_period: 5s
-    command: ["--config=/etc/otel-collector-config.yaml"]
+      start_period: 2s
     ports:
+    - "4137:4137" # gRPC port
```

Comment:
i suspect this isnt required it exposes the port to the host and would usually be referenced in the docs in that case

Reply:
Fair

Reply:
Done in fc93b3573d